framework/include/postprocessors/RealControlParameterReporter.h

```diff
@@ -12,25 +12,25 @@
 /*            See COPYRIGHT for full restrictions               */
 /****************************************************************/
 
-#ifndef REALPARAMETERREPORTER_H
-#define REALPARAMETERREPORTER_H
+#ifndef REALCONTROLPARAMETERREPORTER_H
+#define REALCONTROLPARAMETERREPORTER_H
 
 // MOOSE includes
 #include "GeneralPostprocessor.h"
 #include "ControlInterface.h"
 
 // Forward Declarations
-class RealParameterReporter;
+class RealControlParameterReporter;
 
 template<>
-InputParameters validParams<RealParameterReporter>();
+InputParameters validParams<RealControlParameterReporter>();
 
-class RealParameterReporter :
+class RealControlParameterReporter :
```

Comment:
Is this class really going to be generally useful or is it only for testing?  Maybe it should just go in moose_test